Microsoft.HealthVault.Client.Android/AndroidHealthVaultConnectionFactory.cs

```diff
@@ -32,5 +35,14 @@ public static IHealthVaultConnectionFactory Current
                 }
             }
         }
+
+        public static IThingTypeRegistrar ThingTypeRegistrar
```

Comment:
>public static IThingTypeRegistrar ThingTypeRegistrar [](start = 8, length = 52)

XML docs, same for the other platforms. We may want to consider linking all these identical files to a common source file.